proto/config.proto

```diff
@@ -0,0 +1,59 @@
+package oci;
+	
+// Spec is the base configuration for the container.  It specifies platform
+// independent configuration.
+message Spec {
+	// Version is the version of the specification that is supported.
+  required string Version = 1;
```

Comment:
One thing to consider with `required` fields is that it might be almost impossible to make it optional.